<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(3, 4, 19);
border: 2px solid rgb(121, 121, 121);
}</string>
  </property>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color:  #252984;
color: white;
font: bold 25px;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="HelpText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>561</width>
     <height>230</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit 
{
	background-color: rgb(3, 4, 19);
	border: none;
	color: white;
}

QScrollBar:vertical
{
	background: #1D253D;
	border: 0px solid red;
	border-radius: 7px;
}

QScrollBar::handle:vertical
{
	background: #C6C8CE;
	border: 0px solid red;
	border-radius: 7px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical
{
	height: 0px;
}
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string notr="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin laoreet mi eget sapien molestie, vel semper leo eleifend. Praesent tincidunt quam vel ipsum tempor porttitor. Vestibulum faucibus sem mauris, in tempor dolor iaculis eu. Pellentesque sapien velit, consectetur ac massa id, maximus varius ante. Nam consectetur dolor quis maximus lacinia. Curabitur tempus condimentum nisl sit amet efficitur. Praesent vel leo tincidunt sapien iaculis lacinia eu quis dui.

Vivamus pulvinar dolor non molestie dignissim. Cras a elit nec arcu hendrerit faucibus. Proin sit amet neque mauris. Fusce sit amet enim in lorem tempor scelerisque ut ut lorem. Donec lacinia nulla eu lacus ultricies, quis </string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
